packages/search/src/index.ts

```diff
@@ -103,6 +103,12 @@ const getExportNames = (file: File) => {
     .map(x => (x as any).name);
 };
 
+export const findModuleRoot = async (cwd: string): Promise<string> => {
```

Comment:
This is already a function in `utils/index.ts` 

https://github.com/prodo-ai/snoopy/blob/master/packages/search/src/utils/index.ts#L45

Reply:
Haha fair enough. Good spot!